inc/theme/namespace.php

```diff
@@ -194,6 +194,26 @@ function migrate_book_themes() {
 		$pressbooks_theme_migration = 4;
 		update_option( 'pressbooks_theme_migration', $pressbooks_theme_migration );
 	}
+
+	// Rename Austen Two to Austen
+	if ( $pressbooks_theme_migration === 4 ) {
+		$theme = wp_get_theme()->get_stylesheet();
+		if ( $theme === 'pressbooks-austentwo' ) {
+			// Switch theme to Austen 3.0
+			switch_theme( 'pressbooks-austen' );
+			$lock = Lock::init();
+			if ( $lock->isLocked() ) {
+				$data = $lock->getLockData();
+				$data['stylesheet'] = 'pressbooks-austen';
+				$json = wp_json_encode( $data );
+				$lockfile = $lock->getLockDir() . '/lock.json';
+				\Pressbooks\Utility\put_contents( $lockfile, $json );
+			}
```

Comment:
The `migrate_book_themes` function is starting to get a bit unwieldy. The 8 lines above are, more or less, copy/pasted 3 times now.

On the one hand I think this should be refactored. On the other hand, maybe it's fine because it will only be run once.

What do you think?

Reply:
Probably worth pulling that chunk into its own utility function 👍